tests/Kiota.Builder.Tests/Refiners/PythonLanguageRefinerTests.cs

```diff
@@ -109,37 +109,6 @@ public class PythonLanguageRefinerTests {
 
         Assert.Contains("Error4XX", declaration.Usings.Select(x => x.Declaration?.Name));
     }
-    [Fact]
-    public async Task AddsUsingsForDiscriminatorTypes() {
```

Comment:
Why are these python tests being removed? 
I'm not seeing any other python based changes in this PR?

Reply:
I think they were copy/pasted from the TypeScript tests originally and never updated. If you pay close attention to the body, you'll see this test is actually using the typescript refiner which comforts me in my analysis. Looking at the python refiner, it's not doing any of what's tested by these tests anyway.